specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2020-02-07-preview/examples/SapMonitors_Create.json

```diff
@@ -16,7 +16,8 @@
         "enableCustomerAnalytics": true,
         "logAnalyticsWorkspaceArmId": "",
         "logAnalyticsWorkspaceId": "",
-        "logAnalyticsWorkspaceSharedKey": ""
+        "logAnalyticsWorkspaceSharedKey": "",
```

Comment:
>"" [](start = 42, length = 2)

Can you provide example resource IDs for all the properties that are being specified. `""` is not really helpful to users. (This comment applies to all examples and properties in the examples.)

Reply:
done